BE/api/recipes.js

```diff
@@ -24,7 +24,20 @@ recipesRouter.get("/", async (req, res, next) => {
 recipesRouter.post("/", requireAdmin, async (req, res, next) => {
   const { id: userId } = req.user;
   const { body } = req;
-
+  
+  if(body.tags && body.tags != []){
```

Comment:
I'm not sure this works.

Reply:
why

Reply:
which part? i tested

Reply:
I thought there were issues testing something against a blank array.

Reply:
right now if the recipe has tags, and you edit tags and put in an empty array field, the tags do not update. But that is an extreme edge case and I'm not willing to work on that. This condition to check for blank tags works though.

Reply:
im dumb im editing it